plugin/mouse/mouse.py

```diff
@@ -109,6 +109,11 @@ def gui_wheel(gui: imgui.GUI):
 
 @mod.action_class
 class Actions:
+    def zoom_close():
+        """Closes an in-progress zoom. Talon will move the cursor position but not click."""
+        if eye_zoom_mouse.zoom_mouse.state == eye_zoom_mouse.STATE_OVERLAY:
+            actions.tracking.zoom_cancel()
```

Comment:
Just curious, do we need this wrapper action? What does zoom_cancel do when zoom mouse isn't enabled / or is enabled and idle?

Reply:
probably need to ask @lunixbochs , i just put it in there for safety